tidy3d/components/material/tcad/heat.py

```diff
@@ -69,6 +70,7 @@ class SolidMedium(AbstractHeatMedium):
     """
 
     capacity: pd.PositiveFloat = pd.Field(
+        None,
         title="Heat capacity",
         description=f"Volumetric heat capacity in unit of {SPECIFIC_HEAT_CAPACITY}.",
```

Comment:
I guess now it is not `Volumetric heat capacity` but rather `Specific heat capacity`?